thoth/user_api/api_v1.py

```diff
@@ -264,6 +266,12 @@ def post_advise_python(
     parameters = locals()
     parameters["application_stack"] = parameters["input"].pop("application_stack")
 
+    integrations = [i.value for i in ThothAdviserIntegrationEnum]
+
+    if source_type:
+        if source_type not in integrations:
+            raise NotKnownThothIntegration(f"This integration {source_type} is not provided in Thoth: {integrations}")
```

Comment:
You can put this exception directly to thoth-common to have it as close to error state as possible.